Tests/OfflineTests/DerivationToolTests/DerivationToolMainnetTests.swift

```diff
@@ -53,7 +53,7 @@ class DerivationToolMainnetTests: XCTestCase {
     func testDeriveViewingKeysFromSeed() throws {
         let seedBytes = [UInt8](seedData)
 
-        let spendingKey = try derivationTool.deriveUnifiedSpendingKey(seed: seedBytes, accountIndex: 0)
+        let spendingKey = try derivationTool.deriveUnifiedSpendingKey(seed: seedBytes, account: Account(0))
```

Comment:
```suggestion
        let spendingKey = try derivationTool.deriveUnifiedSpendingKey(seed: seedBytes, accountIndex: Zip32AccountIndex(0))
```
and similarly below.

Reply:
Updated

Reply:
This change doesn't seem to have been applied.

Reply:
Updated, there were 3 more instances of account: Account(0) in Tests so I updated those as well.